modular_skyrat/modules/aesthetics/flag/code/signs_flags.dm

```diff
@@ -0,0 +1,158 @@
+/obj/structure/sign/flag
+	name = "blank flag"
+	desc = "The flag of nothing. It has nothing on it. Magnificient."
+	icon = 'modular_skyrat/modules/aesthetics/flag/icons/flags.dmi'
+	icon_state = "flag_coder"
+	buildable_sign = FALSE
+	custom_materials = null
+	var/item_flag = /obj/item/sign/flag
+
+/obj/structure/sign/flag/wrench_act(mob/living/user, obj/item/wrench/I)
+	return
+
+/obj/structure/sign/flag/welder_act(mob/living/user, obj/item/I)
+	return
+
+/obj/item/sign/flag/welder_act(mob/living/user, obj/item/I)
+	return
```

Comment:
This should be moved lower with all of the other `/obj/item/sign/flag` stuff near the bottom of the file.

Reply:
Will do.